### ISSUE TITLE
Feature: include priority in the csv as well

### ISSUE THREAD

**Issue by tryguy** [Modorganizer2/modorganizer#65]
Currently, exporting to csv gives you nexus_id, name, version and filename.

But, what I'd find more useful is to preserve the listing I have in MO2, namely priority. I don't care how this data is arranged, so long as I can take it and sort it back to the way I see it in MO2.

Currently, it's pretty useless to me, but I can see that info as potentially important. But, the most important is left out. It'd be great if a future version included it.

Actually, category, flags and install date might be useful too. Maybe you could record that in the meta file. It's currently not there, so it'd be something new.

**Comment by twizz0r** [comment]
Also, add Flags, Category and Nexus URL to the CSV export.

Additionally, add a per mod note input that can get exported as well.

Adding more fields to the CSV mod export would make it a stronger tool for authoring mod guides or even just keeping a record of mod organization (and notes if added).

**Comment by Al12rs** [comment]
I added mod priority, primary category, install date and nexus url columns to Export to the csv feature.
I also changed the priority and install date format to allow for better sorting.
The category I could only the primary one. Flags will not tell anything really, the priority is much more indicating of what is going on. To know that a mod is overwriting some file in some other mod higher in the list does not give much info, the priority will do the same wit higher precision. Also adding that would be a pain so I hope you guys can live without it?
